roamance/src/components/social/comment/comment-form.tsx

```
@@ -0,0 +1,320 @@
+                        <motion.div
+                          whileHover={{ scale: 1.05, y: -2 }}
+                          className="relative w-18 h-18 rounded-lg overflow-hidden border border-gray-200/80 dark:border-gray-700/60 group shadow-sm"
+                        >
+                          <Image
+                            src={imagePath}
+                            alt="Uploaded image"
+                            fill
+                            style={{ objectFit: 'cover' }}
+                            className="transition-transform group-hover:scale-110"
+                          />
+                          <div className="absolute inset-0 bg-black/0 group-hover:bg-black/10 transition-colors"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)